pyls_spyder/tests/test_plugin.py

```diff
@@ -26,24 +23,29 @@
 DOC_URI = uris.from_fs_path(__file__)
 DOC = """
 # %%
-# -- Imports
+# ---- Imports
 import os
 import sys
 
-# ------
+# <codecell> Other cell
+# ----
 def a():
+    #### Block comment on a
     # %%% Cell inside a
     for i in range(0, 10):
         # %%%% Cell
         pass
 
 # %%%
 def b():
-    # ----- Pass inside b
+    #---- Pass inside b
     pass
 
+# In[25]
+####
+
 # %% Empty cell
```

Comment:
```suggestion
#%% Empty cell
```

This is to check that we also detect cells when there's no space between `#` and `%%` (which was the main regression reported in #9).